build/sbt

```diff
@@ -94,7 +99,7 @@ process_my_args () {
          -batch) exec </dev/null && shift ;;
 
     -sbt-create) sbt_create=true && shift ;;
-
+    -P*) enableProfile "$1" && shift ;;
```

Comment:
Do we still need this?

This is an orthogonal contribution. If we still need this, we should make an independent PR, @TQJADE .

Reply:
removed